### DISCUSSION TITLE
keep osmand alive when phone is locked.

### DISCUSSION THREAD

**Discussion by SlowJoe** [osmandapp/OsmAnd#13174] [General]
Hi, if you navigate via Google Maps or Komoot and you lock your phone you are able to press the screen lock button once and the navigation shows up again without the need to unlock the phone. This is super useful for longer trips. Would there be the possibility to implement this feature in Osmand? Is it something I can do with tasker other similar apps? How exactly is this feature called?

**Comment by vshcherb** [comment]
Yes, it's present with several options. Please check Settings -> Profile -> General Settings -> Lock Screen.
